src/__tests__/lib/containers/dataaccess/AccessHistoryDashboard.test.tsx

```diff
@@ -179,7 +179,7 @@ describe('AccessHistoryDashboard tests', () => {
     )
 
     userEvent.type(arNameInput, mockAccessRequirement.name)
-    selectEvent.select(
+    await selectEvent.select(
       arNameInput,
       getOptionLabel(mockAccessRequirement.id, mockAccessRequirement.name),
     )
```

Comment:
The `react-select` element asynchronously loads the options from the API, so we should `await` the select event.